assign_rights/templates/groupings/manage.html

```diff
@@ -5,8 +5,15 @@
 
 <form action="." method="POST" accept-charset="utf-8">
   {% csrf_token %}
-	{{ form | crispy }}
+	{{ form.title | as_crispy_field }}
+	{{ form.description | as_crispy_field }}
+	<fieldset>
+		<legend>Choose Associated Rights Shell/s*</legend>
+		<div class="overflow-auto border p-2 mb-3" style="max-height:250px;">
```

Comment:
The way to avoid the inline styles here would be to either add the `max-height` to `.overflow-auto` or to add another class with that style.

Reply:
Right. When I did that I couldn't make it work because of course it was a caching issue. Sigh.